src/sentry/seer/utils.py

```diff
@@ -40,14 +46,7 @@ class BreakpointResponse(TypedDict):
     timeout=settings.ANOMALY_DETECTION_TIMEOUT,
 )
 
-seer_staging_connection_pool = connection_from_url(
-    settings.SEER_AUTOFIX_URL,
-    retries=Retry(
-        total=5,
-        status_forcelist=[408, 429, 502, 503, 504],
-    ),
-    timeout=settings.ANOMALY_DETECTION_TIMEOUT,
-)
+seer_staging_connection_pool = connection_from_url(settings.SEER_AUTOFIX_URL)
```

Comment:
Remove the automatic retries to seer for similarity endpoints

Reply:
do we still want the timeout here?